### DISCUSSION TITLE
Issue Starting Ableton project file

### DISCUSSION THREAD

**Discussion by oliviano** [SudoMagicCode/SudoSignals#17] [v1 - Q&A] [answered]
I have an issue starting an Ableton project file from SudoSignals, Ableton will lunch as intended, but open a new project, rather than the specified one.

If I run a simple .bat file Ableton lunch and opens the project specified.
bat file example:
`start "C:/ProgramData/Ableton/Live 10 Suite/Program/Ableton Live 10 Suite.exe" "%~dp0\assets\audio\riverProject\river.als"`

my yaml config below:

`version: v0.2.5.20210610
installation:
    productid: xxxxxx
    user: ""
startup:
    prestartBAT: ""
    processes:
        - id: 17xxxxxxx
          name: Abelton_10
          programPath: C:\ProgramData\Ableton\Live 10 Suite\Program\Ableton Live 10 Suite.exe
          filePath: D:\TESTProject\TEST.als
          autoStart: true
          autoRestart: true
          restartLimit: 0
          controlled: false
          killTimeout: 0
    poststartBAT: ""
daemon:
    configurationPort: 10000
services: []
`

**Comment by raganmd** [comment]
Thanks for this report @oliviano. We're gonna take a look here and see if we can get to the bottom of what's happening.

**Comment by raganmd** [comment]
Woah Golly - this one really sent us on a goose chase.

The library for starting and stopping apps has a slightly different behavior than the direct command line equivalent. We're currently exploring a few alternatives to help this work smoothly, without breaking other start behaviors. 

The other discover here is that when signals closes Ableton that looks like it's tracked as a crash. So there's likely some clean-up that will be required between restarts. We have a solve here that should work - which is to give a user the ability to run a bat file before all of the process start commands are called. That should give you a way to specify which clean-up scripts you need to run before an app starts. We're currently looking at how we might add that solution.

https://forum.ableton.com/viewtopic.php?t=216566

All to say - no solution yet, but working on finding a resolution to this issue.

**Reply by raganmd** [reply]
One more quick update here @oliviano - it looks like we have a fix that will address this issue. A few more elements to iron out, and we should have a new build of the installer published.

**Comment by oliviano** [comment]
Thanks for the update! Sorry to have send you on a wild Chase.

My colleague Puck Murphy, just flew to Dubai today to install the project
for the project expo, Nz pavilion.

I ll do some studio test here, here. We re running stalker and some old
school bat file for the moment .

Thanks for looking into this.

**Comment by raganmd** [comment, marked as answer]
Alright - we've got a fix up in a new release here:

https://github.com/SudoMagicCode/sudoSignals_releases/releases

few additional fixes, along with Ableton.

The gotcha here is that when Signals quits ableton you may end up with some junk files that Ableton will interpret as a crash. You may still need to run a regular batch script to clean up those junk files from Ableton.
